read_mars/__init__.py

```diff
@@ -42,3 +42,35 @@ def read_mars(filename, tree='Events', verbose=False):
     f.Close()
 
     return pd.DataFrame(events)
+
+
+def read_mars_fast(filename, tree='Events', leaves=[]):
```

Comment:
I created the new function, because I didn't trust the v1 thing at first. But it does work very well for me, so you are right and the old code could be replaced. Maybe you can also try it out.